fn_apility/fn_apility/util/config.py

```diff
@@ -0,0 +1,17 @@
+# (c) Copyright IBM Corp. 2010, 2018. All Rights Reserved.
+# -*- coding: utf-8 -*-
+
+"""Generate a default configuration-file section for fn_apility"""
+
+from __future__ import print_function
+
+
+def config_section_data():
+    """Produce the default configuration section for app.config,
+       when called by `resilient-circuits config [-c|-u]`
+    """
+#    config_data = u"""[fn_apility]
```

Comment:
this should be uncommented and match the parameters you referenced in your code (ex: self.options['url'])

Reply:
Done!